src/test/java/lv/ctco/javaschool/goal/boundary/GoalApiTest.java

```diff
@@ -275,6 +275,82 @@ void testSaveNewCommentsForGoalByIdException() {
         assertThrows(InvalidGoalException.class, () -> goalApi.saveNewCommentsForGoalById(1L, msg));
     }
 
+    @Test
+    @DisplayName("Test if goal is for current user is returned true")
+    void isCurrentUsersGoalTestForCurrentUsersGoal() {
+        when(userStore.getCurrentUser())
+                .thenReturn(user1);
+        when(goalStore.getUserGoalById(user1,1L))
+                .thenReturn(Optional.of(goal));
+        assertThat(goalApi.isCurrentUsersGoal(1L),is(true));
+    }
+
+    @Test
+    @DisplayName("Test if goal is for other user not current, then returned false")
+    void isCurrentUsersGoalTestForOtherUsersGoal() {
+        when(userStore.getCurrentUser())
+                .thenReturn(user2);
+        when(goalStore.getUserGoalById(user2,1L))
+                .thenReturn(Optional.empty());
+
+        assertThat(goalApi.isCurrentUsersGoal(1L),is(false));
+    }
+
+    @Test
+    @DisplayName("Test, that goal is edited, if user created goal")
+    void editGoalTestForGoalOwner() {
+        GoalFormDto goalDto = new GoalFormDto();
+        goalDto.setGoalMessage("123");
+        goalDto.setDeadline(LocalDate.of(2018,1,1));
+
+        when(userStore.getCurrentUser())
+                .thenReturn(user1);
+        when(goalStore.getUserGoalById(user1,1L))
+                .thenReturn(Optional.of(goal));
+
+        goalApi.editGoal(1L,goalDto);
+
+        assertThat(goal.getGoalMessage(),is(goalDto.getGoalMessage()));
+        assertThat(goal.getDeadlineDate(),is(goalDto.getDeadline()));
+    }
+
+    @Test
+    @DisplayName("Test, that goal is edited, if user created goal")
+    void editGoalTestForUserThatIsNotGoalOwner() {
+        GoalFormDto goalDto = new GoalFormDto();
+        goalDto.setGoalMessage("123");
+        goalDto.setDeadline(LocalDate.of(2018,1,1));
+
+        String messageBeforeEdit = goal.getGoalMessage();
+        LocalDate deadlineDateBeforeEdit = goal.getDeadlineDate();
+
+        when(userStore.getCurrentUser())
+                .thenReturn(user2);
+        when(goalStore.getUserGoalById(user2,1L))
+                .thenReturn(Optional.empty());
+
+        goalApi.editGoal(1L,goalDto);
+
+        assertThat(goal.getGoalMessage(),is(messageBeforeEdit));
+        assertThat(goal.getDeadlineDate(),is(deadlineDateBeforeEdit));
+    }
+
+    @Test
+    @DisplayName("Test, that goal is edited, if users new goal is empty")
+    void editGoalTestWithEmptyNewGoalInformation() {
```

Comment:
Please try to reduce the amount of words in method names - e.g. `testEditGoalWithoutInput()` or similar.

Reply:
Renamed test method as suggested.